dexterity/membrane/behavior/user.py

```diff
@@ -164,6 +165,8 @@ def getPropertiesForUser(self, user, request=None):
                 # Would give an error like this:
                 # ValueError: Property home_page: unknown type
                 value = u''
+            if type(value) is RichTextValue:
```

Comment:
Can you change this line to `if isinstance(value, RichTextValue):`?
I will merge this PR anyway but isinstance should the way to here: https://stackoverflow.com/questions/1549801/what-are-the-differences-between-type-and-isinstance